### ISSUE TITLE
Tracking issue: Load default presets

### ISSUE THREAD

**Issue by tomasciccola** [digidem/mapeo-core-next#395]
## Description

When a project is created the user should get the default presets with its associated fields and icons. What we end up after talking with @gmaclennan is the following:
1. Bundle the default config file inside the app (probably on this repo). This can change in terms of user need. i.e. the config could be distributed as a regular file that gets loaded through a dialog in the frontent.
2. When creating a new project, we first delete all the presets in the db (to avoid db bloat) and then load and write to the db the default fields, icons and lastly the presets.
3. After inviting a new peer to the project, they would get all the default presets from the project creator

Additionally we discussed replacing the format from a tarball to a zip (since it can be open in every OS without a command line) and the extension from `.mapeosettings` to `.mapeoconfig`.
 I've already created the issue on the [mapeo-settings-builder repo](https://github.com/digidem/mapeo-settings-builder/issues/50) to take that into account for future configs

- [x] #401 
- [x] #402

**Comment by gmaclennan** [comment]
Can you split this into two issues:

1. Add a `project.importConfig({ pathToConfigFile })` method that imports presets, fields and icons from a config file into the database.
2. Pre-populate the database with default config when a project is first created.

(1) should be done first, then we can work on (2) separately.
